### PR TITLE
Add Spanish to the list of default languages

### PR DESCRIPTION
**Pull request by mikhailshilkov**
Add Spanish to the list of default languages for relative time plugin

### CODE REVIEW

File: dist/amd/defaultTranslations/relative.time.js [hunk 1]
```diff
@@ -55,6 +55,35 @@ define(['exports'], function (exports) {
         'year_in_plural': 'in __count__ years'
       }
     },
+    es: {
```
Comment (atsu85) [inline dist/amd/defaultTranslations/relative.time.js]: please revert the changes in generated files (under dist folder) - they are updated with release process.

Comment (atsu85) [inline dist/amd/defaultTranslations/relative.time.js]: @EisenbergEffect, another PR in short time with changes in dist - what do You think about the git hook i proposed previously to check that files under dist folder aren't included in non-release commit?

Comment (mikhailshilkov) [inline dist/amd/defaultTranslations/relative.time.js]: @atsu85 reverted dist

**Comment by zewa666** [comment]
@mikhailshilkov thanks for the contribution and thanks @atsu85 for pointing out not to include the dist folder. @EisenbergEffect I agree that a hook, if easy to be implemented would be nice to check for any occurrence of dist included files
